cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -56,14 +57,24 @@ open class Node @Inject constructor(private val project: Project) {
         private set
     private var rpcSettings: RpcSettings = RpcSettings()
     private var webserverJar: String? = null
-    private var p2pPort = 10002
+    internal var p2pPort = 10002
+        @Input get
     internal var rpcPort = 10003
         @Input get
         private set
     internal var config = ConfigFactory.empty()
         @Internal get
         private set
 
+    lateinit var dbSettings: DbSettings
```

Comment:
This should also be `private set` I think. Does it need to be `lateinit`, or do we know what the `DbSettings` are going to be when the class is created?